src/pages/Users/Users.tsx

```diff
@@ -1,25 +1,29 @@
+import { Outlet, useLoaderData } from 'react-router-dom'
 import { UniversalTable, AddButton } from '@/components'
-import { UsersLoaderData, User } from './types'
+import { mapDataUserPage, UserData } from './helper'
+import { ADMIN_ROUTES_NAMES } from '@/routing/routes.names'
 import classes from './Users.module.scss'
-import { useLoaderData } from 'react-router-dom'
 
 const Users = () => {
-	const { header, tableData } = (useLoaderData() as UsersLoaderData<User>) || {}
+	const usersDataResponse = useLoaderData() as UserData[]
+	const tableData = mapDataUserPage(usersDataResponse)
+
 	return (
 		<>
+			<Outlet />
 			<div className={classes.usersPage}>
 				<div className={classes.usersPageOptions}>
-					<AddButton action={() => console.log('Кніпка натиснена')} />
-					<h2 className={classes.usersPageOptionsHeader}>{header}</h2>
-				</div>
-				{tableData && tableData.data && tableData.headers ? (
```

Comment:
are you sure that we don't need to check wether the data is there?
bring beck the checking and show a message when there's no data